README.md

```diff
@@ -54,40 +71,193 @@ resource "aws_ami_from_instance" "example" {
 }
 ```
 
-## Variables
-
-|  Name                           |  Default       |  Description                                                                                  | Required|
-|:--------------------------------|:--------------:|:----------------------------------------------------------------------------------------------|:-------:|
-| `namespace`                     | `global`       | Namespace (e.g. `cp` or `cloudposse`) - required for `terraform-null-label` module            | Yes     |
-| `stage`                         | `default`      | Stage (e.g. `prod`, `dev`, `staging` - required for `terraform-null-label` module             | Yes     |
-| `name`                          | `admin`        | Name  (e.g. `bastion` or `db`) - required for `terraform-null-label` module                   | Yes     |
-| `ec2_ami`                       | `ami-cd0f5cb6` | By default it is an AMI provided by Amazon with Ubuntu 16.04                                  | No      |
-| `ssh_key_pair`                  | ``             | SSH key pair to be provisioned on instance                                                    | Yes     |
-| `github_api_token`              | ``             | GitHub API token                                                                              | Yes     |
-| `github_organization`           | ``             | GitHub organization name                                                                      | Yes     |
-| `github_team`                   | ``             | GitHub team                                                                                   | Yes     |
-| `instance_type`                 | `t2.micro`     | The type of instance that will be created (e.g. `t2.micro`)                                   | No      |
-| `vpc_id`                        | ``             | The ID of the VPC where the instance will be created                                          | Yes     |
-| `security_groups`               | []             | List of Security Group IDs permitted to connect to this instance                              | Yes     |
-| `subnets`                       | []             | List of VPC Subnet IDs where the instance may be launched                                     | Yes     |
-| `allow_cidr_blocks`             | ["0.0.0.0/0"]  | List of CIDR blocks to permit SSH access                                                      | No      |
-| `zone_id`                       | ``             | ID of the domain zone to use - is a result of terraform-aws-route53-cluster-zone output       | Yes     |
+
+
+
+
+
+## Makefile Targets
+```
+Available targets:
+
+  help                                This help screen
+  help/all                            Display help for all targets
+  lint                                Lint terraform code
+
+```
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| allow_cidr_blocks |  | list | `<list>` | no |
```

Comment:
Missing descriptions for all inputs. We can borrow most of them from the ec2 instance module